sdk/daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Upgrading.scala

```diff
@@ -442,7 +448,7 @@ case class TypecheckUpgrades(
 
     val moduleWithMetadata = module.map(ModuleWithMetadata)
     for {
-      (existingTemplates, _new) <- checkDeleted(
+      (existingTemplates, newTemplates) <- checkDeleted(
```

Comment:
no pattern matching in the left of `<-` .
https://discuss.daml.com/t/pattern-always-adds-withfilter-scala-ritual/1463